pkg/secrets/secrets.go

```diff
@@ -80,3 +85,69 @@ func GenerateUserData(publicKey ssh.PublicKey) (*core.Secret, error) {
 
 	return userDataSecret, nil
 }
+
+// Encrypt creates an ecnrypted block of text from a plaintext message using the given key
+func Encrypt(plainText string, key []byte) (string, error) {
+	if key == nil {
```

Comment:
is nil an adequate check for empty bytes?

Reply:
Empty bytes should be fine, covered in [unit tests](https://github.com/openshift/windows-machine-config-operator/pull/508/commits/5004d8efe302d755d5f0812a8472b6d20771af8a#diff-0be37f7149d650a2c98fc7c079b136182f607b5e2f98587dce5f3f00657d38b7R86-R92)